host-controller/src/main/java/org/jboss/as/domain/controller/operations/HostProcessReloadHandler.java

```diff
@@ -76,10 +76,13 @@
             .setAlternatives(ModelDescriptionConstants.USE_CURRENT_DOMAIN_CONFIG)
             .build();
 
+    protected static final AttributeDefinition TIMEOUT = new SimpleAttributeDefinitionBuilder(ModelDescriptionConstants.TIMEOUT, ModelType.INT, true)
+            .setDefaultValue(new ModelNode(0)).build();
```

Comment:
WFCORE-1616 has been fixed; see #1630. So now a call that hits ServerSuspendHandler will not complete until the suspend is done.